.changeset/short-ligers-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": patch
+---
+
+fix(blade): checkbox icon wrapper position
```

Comment:
Changelog should also say what bug it fixes. Makes it easier to understand from reader's perspective imo.

Reply:
Done!